src/example_fgen_basic/error_v/creation_wrapper.f90

```diff
@@ -25,13 +25,12 @@ module m_error_v_creation_w
 contains
 
     function create_error(inv) result(res_instance_index)
-        !! Wrapper around `m_error_v_creation.create_error` (TODO: x-ref)
+        !> Wrapper around `m_error_v_creation.create_error` ([[m_error_v_creation(module):create_error(function)]])
```

Comment:
```suggestion
        !! Wrapper around `m_error_v_creation.create_error` ([[m_error_v_creation(module):create_error(function)]])
```

`!>` means document the thing below this. `!!` means document the thing above this. (this is mentioned somewhere in the Ford docs)

this is why the docs here for `inv` are wrong: https://example-fgen-basic--32.org.readthedocs.build/en/32/fortran-api/proc/create_error~2.html